public/scripts/app.js

```diff
@@ -36,61 +45,145 @@ function setCurrentSpan(ele, view) {
 		$(ele).find('span').text('Current');
 	} else {
 		$(ele).find('span').text('');
-	}	
+	}
 }
 
 function renderAddClassBtn(view) {
-	var titleStuff = '.title-stuff';
-	$(titleStuff).empty();
+	var titleArea = '.title-area';
+	$(titleArea).empty();
 	if (state.viewProps[view].addClassBtn) {
-		$(titleStuff).append('<a>Add a Class</a>');
-		$(titleStuff)
+		$(titleArea)
+		.append('<h1>Student Grades App</h1>')
+		.append('<a>Add A Class</a>');
+
+		$(titleArea)
 		.find('a')
 		.attr('role', 'button')
 		.attr('href', "/classes/class/create")
 		.attr('id', 'js-add-class-btn')
-		.addClass('btn')
-		.addClass('btn-danger')
 		.addClass('add-class-btn');
 	} else {
-		$(titleStuff)
+		$(titleArea)
 		.find('a')
 		.remove();
 	}
 }
 
-function drawLbButtons(value, style, actn, type, txt) {
+function drawLbButtons(value, actn, type, txt) {
 	let ele = '#' + value;
 	$(ele)
 	.append('<button></button>');
 
 	$(ele)
 	.find('button')
 	.attr('type', type)
-	.addClass('btn')
-	.addClass('btn-' + style)
 	.addClass('js-' + actn + '-trigger')
 	.text(txt);
 }
 
-function drawAnchorButton(text, href, style, value) {
+function drawAnchorButton(text, href, value) {
 	var ele = '#' + value,
 		fullHref = href + value;
 
-	$(ele).append('<a></a>');
+	$(ele).append('<td></td>');
 
 	$(ele)
+	.find('td')
+	.last()
+	.addClass('hidden')
+	.addClass(text.toLowerCase())
+	.append('<a></a>')
 	.find('a')
 	.last()
 	.attr('role', 'button')
-	.attr('id', value)
+	.attr('data', value)
 	.addClass('class-finder')
-	.addClass('btn')
-	.addClass(style)
 	.text(text)
 	.attr('href', fullHref);
 }
 
+function drawDelButton(value) {
+	var ele = '#' + value;
+
+	$(ele).append('<td></td>');
+
+	$(ele).find('td')
+	.last()
+	.addClass('hidden')
+	.addClass('delete')
+	.append('<button></button>')
+	.find('button')
+	.attr('type', 'button')
+	.attr("data", value)
+	.addClass('js-send-to-del-trigger')
+	.text('Delete');
+}
+
+function handleDropDownClick(index) {
+	$('.js-dropdown-btn-' + index).click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-' + index).toggleClass('show');
+	});
+
+	$(window).click(function(e) {
```

Comment:
Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu